Xamarin.Forms.Core/Items/IItemsViewSource.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel;
+
+namespace Xamarin.Forms
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Is this being used by anything on the Forms side? Or is it just here to avoid duplicating code?

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/5044/files#diff-a71d7c0190ea452b24b7a49a4fae2a80R73